libopenage/util/fixed_point_test.cpp

```diff
@@ -127,7 +127,8 @@ void fixed_point() {
 		using T = FixedPoint<uint16_t, 7U, uint64_t>;
 
 		auto a = S::from_int(16U);
-		TESTNOTEQUALS((a*a).to_int(), 256U);
+		// This test case is now equal with the improved multiplication algorithm
```

Comment:
You should probably remove the comment since no one will know what "improved multiplication algorithm" refers to in a few months :D